libs/js-sdk/src/client.ts

```diff
@@ -65,9 +63,10 @@ export function createBrowserClient({
 
     // Ensure that the init promise doesn't cause an unhandled promise rejection
     initialisedState.initialisedPromise.promise.catch(() => {})
+
     const state = new EffectiveFeaturesState(
```

Comment:
I wonder if we should collapse `EffectiveFeaturesState` and `EffectiveFeaturesStore` into a single class, and remove the interfaces?

Reply:
I believe we could do that, there is not really any reason to keep them apart when we removed the ability set a store.